### ISSUE TITLE
Add questionnaire for accessibility rating

### ISSUE THREAD

**Issue by holgerd** [sozialhelden/wheelmap-classic#466]
In order to increase the quality of the wheelchair ratings, as a newbie on Wheelmap I want to answer concrete questions about the accessibility of the place I'm reviewing so I can give precise answers which really help wheelchair drivers understand the accessibility of said place.

Current form:
![wheelmap_-_find_wheelchair_accessible_places](https://cloud.githubusercontent.com/assets/218506/19834749/069a825c-9e6d-11e6-94ba-9ba47e24a11c.png)
# Rough idea of a newbie user journey:
1. select place I want to review and oben the rating dialogue:
2. Instead of the red/yellow/green select boxes you see "Rate this place now"
   - Open question: Will this lower the number of ratings we get?
3. When clicked, a questionnaire starts and asks one question at a time.
   - Also show a status indicator (eg. "Q 2 of 5")
4. In the last step the user sees her input and how this resulted in a yes/no/limited rating. Then the user can click "save".
   - There is also a checkbox "Let me mark places directly in the future" which will prevent showing all questions in the future for this user. It will look like it is now then (the traffic light select boxes).
   - How can s/he undo this decision later? Delete cookie?
## Ideas for questionnaire:
- "Is this location really a place that is open to the public?" YES/NO
  - If NO: "Please describe what's incorrect about this place" (textarea, create [OSM Note](https://wiki.openstreetmap.org/wiki/Notes))
- "Does the entrance at the location have any steps?"
  - If YES: "Is the step or stairway at the entrance higher than 7cm?"
- "Inside the location, are there steps (separating rooms or levels)?"
- "Does this location have a wheelchair accessible toilet?"
- "Is the name of the location correct as indicated?"
  - If NO: "What is the current name of the location? Please try to use proper punctuation
    and capitalization." (input, create [OSM Note](https://wiki.openstreetmap.org/wiki/Notes))
- "Is the type of place really a [node_type name here]?" (eg. restaurant)
  - If NO: "What is the current type of this place?" (choose from dropdown + "other", create [OSM Note](https://wiki.openstreetmap.org/wiki/Notes))

Another idea:
- Please take at least 2 photos of the location. The first photo should clearly show the entrance and any possible barriers (steps, narrow doorways, etc.). The second photo should show the store's sign / logo with its name and/or street number.

**Comment by Hoverbear** [comment]
Similar to what Google Maps does? Ex:

![screenshot_20161030-155621](https://cloud.githubusercontent.com/assets/130903/19837422/845e7800-9eb9-11e6-9a1e-c48fc0beefd0.png)

**Comment by holgerd** [comment]
@Hoverbear yes, but on a more detailed level regarding accessibility. What's also an open question here: Shall we save each answer or just use it to find the correct wheelchair=yes/no/limited rating.

**Comment by schultyy** [comment]
Saving the answers might also help to provide a better quality of mappings in the future. At least we could see if the answers for this place are consistent or if people have different opinions on the place's accessibility.

**Comment by holgerd** [comment]
We plan a design/concept workshop on Feb 21 together with @pixtur

**Comment by holgerd** [comment]
For https://accessiblebath.org/ somebody created this awesome flow chart:
https://www.bathhacked.org/wp-content/uploads/2017/05/mapping-flowchart.pdf
